common/common.go

```diff
@@ -12,6 +20,38 @@ const (
 	TEST_NETWORK
 )
 
+var (
+	// Pegnet Burn Addresses
+	BurnAddresses = map[string]string{
+		"main": "EC1moooFCT2mainoooo1oooo1oooo1oooo1oooo1oooo1opfDJqF",
+		"test": "EC1moooFCT2TESToooo1oooo1oooo1oooo1oooo1oooo1on1iNDk",
+	}
+)
+
+func PegnetBurnAddress(network string) string {
+	return BurnAddresses[strings.ToLower(network)]
+}
+
+// TODO: Remove this, just making sure w/e burn address we come up with is valid
```

Comment:
Usually when reading a "TODO: remove" my immediate question is when? Describe the condition that allows to safely remove that. Otherwise you can bet that no-one will ever take the risk to remove it by fear of breaking something unexpected.

Reply:
Done :+1: